sdcm/provision/azure/virtual_machine_provider.py

```diff
@@ -168,6 +168,7 @@ def add_tags(self, name: str, tags: Dict[str, str]) -> VirtualMachine:
     def _get_os_profile(computer_name: str, admin_username: str,
                         admin_password: str, ssh_public_key: str, custom_data: str) -> Dict[str, Any]:
         os_profile = {"os_profile": {
+            "allowExtensionOperations": False,
```

Comment:
I'm not sure if it will work - other's are in snake_case: `allow_extension_operations`

Reply:
as if Azure docs were that clear about it 😭

Reply:
after digging a bit more, I've found:
https://learn.microsoft.com/en-us/python/api/azure-mgmt-compute/azure.mgmt.compute.v2021_03_01.models.osprofile?view=azure-python#parameters

Reply:
fixed